ci/run.sh

```diff
@@ -98,7 +98,7 @@ function gg_run {
 function gg_run_ctest_debug {
     cd ${SRC}
 
-    rm -rf build-ci-debug && mkdir build-ci-debug && cd build-ci-debug
+    mkdir -p build-ci-debug && cd build-ci-debug
```

Comment:
Let's keep the `rm -rf` version because I want to start from a clean build folder each time

Reply:
Yes, I will restore it.